src/api/app/services/scm_status_reporter.rb

```diff
@@ -8,18 +8,7 @@ def initialize(payload, scm_token, event = nil)
     # Depending on the SCM, it's different
```

Comment:
This comment and the 2 following comment lines should also be moved to the new method.

Reply:
done